File: CPP/Program/Tv.cpp
#include <iostream>
#include <string>
#include "Produk.cpp"
using namespace std;

class Tv : public Produk
{

private:
    int UkuranLayar;
    string resolusi;
    string tipeLayar;

public:
    Tv()
    {
    }
    //constructor
    Tv(int id_produk, string merk, int harga, int stok, int UkuranLayar, string resolusi, string tipeLayar) : Produk(id_produk, merk, harga, stok)
    {

        this->UkuranLayar = UkuranLayar;
        this->resolusi = resolusi;
        this->tipeLayar = tipeLayar;
    }

    //getter dan setter
    int getUkuranLayar()
    {
        return UkuranLayar;
    }

    void setUkuranLayar(int UkuranLayar)
    {
        this->UkuranLayar = UkuranLayar;
    }

    string getResolusi()
    {
        return resolusi;
    }

    void setResolusi(string resolusi)
    {
        this->resolusi = resolusi;
    }

    string getTipe()
    {
        return tipeLayar;
    }

    void setTipe(string tipeLayar)
    {
        this->tipeLayar = tipeLayar;
    }
};

File: PYTHON/Program/Tv.py
from Produk import Produk  

class Tv(Produk):
    #constructor
    def __init__(self, id_produk, merk, harga, stok, UkuranLayar, resolusi, TipeLayar):
        super().__init__(id_produk, merk, harga, stok)
        self.UkuranLayar = UkuranLayar
        self.resolusi = resolusi
        self.tipeLayar = TipeLayar

    # getter dan setter
    def getUkuranLayar(self):
        return self.UkuranLayar

    def setUkuranLayar(self, UkuranLayar):
        self.UkuranLayar = UkuranLayar

    def getResolusi(self):
        return self.resolusi

    def setResolusi(self, resolusi):
        self.resolusi = resolusi

    def getTipe(self):
        return self.tipeLayar

    def setTipe(self, tipeLayar):
        self.tipeLayar = tipeLayar


